### ISSUE TITLE
[datalake-filesystem] Move the operation constants to specific functions.

### ISSUE THREAD

**Issue by afsalthaj** [Azure/azure-sdk-for-go#905]
Currently, the operation constants to do (webhdfs) filesystem operation are being accepted as function arguments, which leads to potential errors on the client side.

**Example**: GetACLStatus function accepts any string as operation constant.

**Fix/Improvement suggestion**:

 Instead of 
```
func (client GroupClient) GetACLStatus(accountName string, pathParameter string, **op string**, tooID *bool) (result ACLStatusResult, err error) {
	req, err := client.GetACLStatusPreparer(accountName, pathParameter, **op**, tooID)
	if err != nil {
```

We could have:

```
func (client GroupClient) GetACLStatus(accountName string, pathParameter string, tooID *bool) (result ACLStatusResult, err error) {
	req, err := client.GetACLStatusPreparer(accountName, pathParameter, **"GETACLSTATUS"**, tooID)
	if err != nil {
```

Currently, if we call`GetACLStatus` with op=LISTSTATUS it results in list of FileStatusResult (which obviously leads to JSON Unmarshal errors) instead of ACLStatusResult.

**Comment by jhendrixMSFT** [comment]
This looks to be a bug in the Go generator, a required single-value enum should be treated as a constant.

**Comment by jhendrixMSFT** [comment]
Fixed in v14.0.0.
